local/Dockerfile.db

```diff
@@ -0,0 +1,4 @@
+FROM ghcr.io/vincentsarago/postgis:14-3.3
+
+#sample data
+COPY data/*.sql /docker-entrypoint-initdb.d/
```

Comment:
`docker compose up` fails for me on this line - is this a feature to seed a dev database, or is there a missing file in the PR?

Reply:
Yes that's just to include some sample data in the database. You an either comment out line 4 or include a `data` folder with some sample data. Can use this and try: https://github.com/developmentseed/tifeatures/tree/master/data

Reply:
I removed COPY data. For local dev, we can load data using ogr2ogr